runtime/float32.c

```diff
@@ -311,51 +311,87 @@ CAMLprim value caml_ldexp_float32_bytecode(value f, value i)
   return caml_copy_float32(caml_ldexp_float32(Float32_val(f), Int_val(i)));
 }
 
-float caml_sse_float32_min(float x, float y) {
+float caml_simd_float32_min(float x, float y) {
```

Comment:
Are the `caml_sse` names still defined anywhere? 
I think we should require that all `caml_[arch}` symbols are defined in the `ocaml_simd` libraries instead of the runtime, but there could be existing code that uses the symbols directly, and needs to link against them even though they're @@bulitin.
This should be easy to fix though; I think there's only a couple places in base/float32 that do this.

Reply:
> Are the caml_sse names still defined anywhere?
only the rounding one right now, which I'll be removing in a moment.